mm2src/coins/utxo/bch.rs

```diff
@@ -1016,6 +988,10 @@ impl SwapOps for BchCoin {
         utxo_common::search_for_swap_tx_spend_other(self, input, utxo_common::DEFAULT_SWAP_VOUT).await
     }
 
+    fn check_all_inputs_signed_by_pub(&self, tx: &[u8], expected_pub: &[u8]) -> Result<bool, String> {
```

Comment:
Let's inline this function

Reply:
I just checked, and it is written that the general rule is that if a function is short, e.g. consists of only one line, then it can be inlined. So I guess in addition to the `check_all_inputs_signed_by_pub` I can as well inline all the functions in the SwapOps?

Reply:
> I can as well inline all the functions in the SwapOps?

Yes, we should inline those as well. As a first iteration I didn't review the PR copletely so I haven't seen those I guess.